.github/workflows/stage.yml

```diff
@@ -53,17 +53,19 @@ jobs:
         run: |
           python3 -m pip install docker
 
-      - name: Log in to GHCR
-        run: |
-          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
 
-      - name: Build and stage awx-operator
-        working-directory: awx-operator
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+
+      - name: Stage awx-operator
         run: |
-          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.default_awx_version }} \
```

Comment:
@jon-nfc Unfortunately, since we need to be able to pin the version of the awx image at build time of the operator, that kind poses a problem for this approach of using the pre-built images from `devel`, re-tagging, and releasing them...

Reply:
the origin of the "known" image in the chain I think can still be maintained by keeping the dev build as that, dev/testing; however independent of the release cycle. The release image chain could be started from the `stage` action and then remain the same as it appears this is where the version number is added within the workflow.

#### Current HEAD

https://github.com/ansible/awx-operator/blob/07427be0b79411f5c8fd2ffd7caf4fd92fcce47a/Dockerfile#L1-L25

https://github.com/ansible/awx-operator/blob/07427be0b79411f5c8fd2ffd7caf4fd92fcce47a/.github/workflows/devel.yaml#L1-L27

https://github.com/ansible/awx-operator/blob/07427be0b79411f5c8fd2ffd7caf4fd92fcce47a/.github/workflows/stage.yml#L1-L85